src/components/Editor/EditorMenuBar.tsx

```diff
@@ -0,0 +1,185 @@
+import { Editor } from "@tiptap/react";
+import {
+  Bold,
+  Code,
+  CodeSquare,
+  Heading1,
+  Heading2,
+  Heading3,
+  Heading4,
+  Heading5,
+  Heading6,
+  Italic,
+  List,
+  ListOrdered,
+  Quote,
+  Redo,
+  Strikethrough,
+  Undo,
+} from "lucide-react";
+import { Codepen } from "react-feather";
+
+const EditorMenubar = ({ editor }: { editor: Editor }) => {
+  return (
+    <div className="flex flex-wrap gap-2">
```

Comment:
Instead of different button components, can we configure a local dictionary and based on that the buttons will get generated? I think it will be a good idea

Reply:
I tried it - there seem to be a issue with types upon importing functions on a local directory . I'll reach out to mentor to see I could solve this on discord - I tried looking at google and couldn't find the solution